### ISSUE TITLE
Acceleration limit constraint

### ISSUE THREAD

**Issue by ahundt** [jrl-umi3218/Tasks#27]
In addition to the position/velocity constraints, is there an acceleration constraint available? Presumably it should be fairly trivial to implement if not since everything is done in acceleration space.

**Comment by haudren** [comment]
Just to be clear, you do mean a joint acceleration constraint? 

If so, there is nothing implemented for now. Indeed, we usually prefer to set a torque limit. But yes, a joint acceleration constraint would be easy to implement.

**Comment by jovenagravante** [comment]
Small thing to add: the joint limits (torque, position, velocity) also correspond nicely to the URDF limit tag so these values are somewhat standard/easy to get.

**Comment by ahundt** [comment]
Yes sorry I meant a constraint. Right now we are controlling our robot via position only but the system has built in acceleration limits so we want to keep the rate of motion to within those limits.

**Comment by ahundt** [comment]
I'm looking into implementing this, would it be best to extend [BoundedSpeedConstr](https://github.com/jrl-umi3218/Tasks/blob/cbfd17ce8b96968978ad958b620d6abdcf25647e/src/QPConstr.h#L529) with acceleration and name it BoundedCartesianMotionConstr, or should I create a separate BoundedAccelConstr?

**Comment by ahundt** [comment]
I went with just one object, seemed simpler and like it will make for less duplicate code in #28.

I'd appreciate a review, still need to make the unit test. It is on top of the alignment changes to avoid those crashes.
